### PR TITLE
Updating top level README

### PR DESCRIPTION
**Pull request by kfswain**
Adding a tad more details to the README, should potential contributors or interested parties land on the repo first.

### CODE REVIEW

**Comment by kfswain** [comment]
@robscott Kept this short and sweet, expecting that we update this as we develop. PTAL, thanks!

File: README.md [hunk 1]
```diff
@@ -1,29 +1,23 @@
...
-- a `README.md` outlining the project goals, sponsoring sig, and community contact information
-- an `OWNERS` with the project leads listed as approvers ([docs on `OWNERS` files][owners])
-- a `CONTRIBUTING.md` outlining how to contribute to the project
-- an unmodified copy of `code-of-conduct.md` from this repo, which outlines community behavior and the consequences of breaking the code
-- a `LICENSE` which must be Apache 2.0 for code projects, or [Creative Commons 4.0] for documentation repositories, without any custom content
-- a `SECURITY_CONTACTS` with the contact points for the Product Security Team 
-  to reach out to for triaging and handling of incoming issues. They must agree to abide by the
-  [Embargo Policy](https://git.k8s.io/security/private-distributors-list.md#embargo-policy)
-  and will be removed and replaced if they violate that agreement.
+This Gateway is intented provide value to multiplexed LLM services on a shared pool of compute. See the [proposal](https://github.com/kubernetes-sigs/wg-serving/tree/main/proposals/012-llm-instance-gateway) for more info.
```
Comment (ahg-g) [inline README.md:5]:
```suggestion
This Gateway is intended to provide value to multiplexed LLM services on a shared pool of compute. See the [proposal](https://github.com/kubernetes-sigs/wg-serving/tree/main/proposals/012-llm-instance-gateway) for more info.
```

File: README.md [hunk 2]
```diff
@@ -1,29 +1,23 @@
-# Kubernetes Template Project
+# Kubernetes LLM Instance Gateway
 
-The Kubernetes Template Project is a template for starting new projects in the GitHub organizations owned by Kubernetes. All Kubernetes projects, at minimum, must have the following files:
+The LLM Instance Gateway is a part of [wg-serving](https://github.com/kubernetes/community/tree/master/wg-serving), and this repo contains: the load balancing algorithm, [ext-proc](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/ext_proc_filter) code, CRDs, and controllers to support the Instance Gateway.
```
Comment (terrytangyuan) [inline README.md:3]:
```suggestion
The LLM Instance Gateway is a part of [wg-serving](https://github.com/kubernetes/community/tree/master/wg-serving), and this repo contains: the load balancing algorithm, [ext-proc](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/ext_proc_filter) code, CRDs, and controllers to support the LLM Instance Gateway.
```

File: README.md [hunk 3]
```diff
@@ -1,29 +1,23 @@
...
-You can reach the maintainers of this project at:
+For more rapid testing, our PoC is in the `./examples/` dir.
 
-- [Slack](https://slack.k8s.io/)
-- [Mailing List](https://groups.google.com/a/kubernetes.io/g/dev)
+## Contributing
+
+Our community meeting is weekly at Th 10AM PDT; [zoom link here](https://zoom.us/j/9955436256?pwd=Z2FQWU1jeDZkVC9RRTN4TlZyZTBHZz09).
+
+We currently utilize the [#wg-serving](https://kubernetes.slack.com/?redir=%2Fmessages%2Fwg-serving) slack channel for comms.
```
Comment (terrytangyuan) [inline README.md:17]:
```suggestion
We currently utilize the [#wg-serving](https://kubernetes.slack.com/?redir=%2Fmessages%2Fwg-serving) slack channel for communications.
```

**Comment by Xunzhuo** [comment]
LGTM, besides the nits pointed out from others.

**Comment by Joffref** [comment]
LGTM

**Review by terrytangyuan — APPROVED** [review]
/lgtm
/approve

**Pull request merged**